.github/workflows/github-ci.yml

```diff
@@ -272,8 +272,27 @@ jobs:
         path: |
           vlsvextract_DP
           vlsvdiff_DP
+          fluxfunction
         if-no-files-found: error
 
+  test_flixfunction:
+    runs-on: carrington
+    needs: [build_tools]
+
+    steps:
+    - name: Download fluxfunction binary
+      uses: actions/download-artifact@v4
+      with:
+        name: vlasiator-tools
+    - name: Make binary executable
+      run: |
+        chmod +x $GITHUB_WORKSPACE/fluxfunction
+    - name: Run with a sample file and compare output
+      run: |
+        $GITHUB_WORKSPACE/fluxfunction /wrk-vakka/group/spacephysics/vlasiator/2D/ABC/bulk/bulk.0001000.vlsv $GITHUB_WORKSPACE/fluxtest.bin
```

Comment:
Noo, just use some of the testpackage file, Magnetosphere_small or Magnetosphere_polar_small.

Reply:
But then we actually need to upload them as artefacts and download them again on the other side. Nobody guarantees that the separate job steps are run with the same github runner.
And since this is run on carrington anyway: why not use a file that's available to all runners equally? The "reference data" here shouldn't change ever.

Reply:
But I'm hardly interested in current support of producing fluxfunction files for a run from a decade ago. What I'm rather more interested in is the bleeding edge, so I can promise local or remote collaborators they can compile this and run it over their fresh files and obtain usable files. :) Without having to spend a day or week fixing that tool again. :)

Reply:
Ah, now that's a good point. I picked ABC since it had been lying around for a decade, and I assumed it would be for another one at least... 
So what's a good current run that we'll probably keep around for a long while? FID?

Reply:
Ah, a good current *2D* run that we'll keep around.